dwave/optimization/include/dwave-optimization/graph.hpp

```diff
@@ -234,6 +248,7 @@ class Node {
     friend void Graph::reset_topological_sort();
     template <class NodeType, class... Args>
     friend NodeType* Graph::emplace_node(Args&&... args);
+    friend ssize_t Graph::remove_unused_nodes(bool);
```

Comment:
IMO it's nice to add the names of the arguments in declaration even though it's not necessary

Reply:
In the context of a `friend` declaration it seems even less relevant, but can do.

Reply:
Fair enough!